backend/canisters/user/impl/src/updates/send_message.rs

```diff
@@ -10,10 +10,13 @@ use canister_api_macros::update;
 use canister_tracing_macros::trace;
 use chat_events::EditMessageArgs;
 use chat_events::EditMessageResult;
+use chat_events::TextContentInternal;
 use chat_events::{MessageContentInternal, PushMessageArgs, Reader, ReplyContextInternal, ValidateNewMessageContentResult};
 use constants::{MEMO_MESSAGE, OPENCHAT_BOT_USER_ID};
+use event_store_producer::NullRuntime;
 use event_store_producer_cdk_runtime::CdkRuntime;
 use rand::Rng;
+use std::ops::Not;
```

Comment:
This is probably not meant to be here

Reply:
It is used on line 361

Reply:
Ahh right it enables the `.not()` method.